ProcessesApi.Tests/V1/Gateways/ProcessesGatewayTest.cs

```diff
@@ -156,23 +162,60 @@ public async Task UpdateProcessByIdSuccessfullySaves()
-            response.CurrentState.Assignment.Should().BeEquivalentTo(updateProcessRequest.Assignment);
-            response.CurrentState.UpdatedAt.Should().BeCloseTo(DateTime.UtcNow, 2000);
-            response.VersionNumber.Should().Be(1);
+            load.CurrentState.ProcessData.FormData.Should().BeEquivalentTo(updateProcessRequest.ProcessData.FormData);
+            load.CurrentState.ProcessData.Documents.Should().BeEquivalentTo(updateProcessRequest.ProcessData.Documents);
+            load.CurrentState.Assignment.Should().BeEquivalentTo(updateProcessRequest.Assignment);
+            load.CurrentState.UpdatedAt.Should().BeCloseTo(DateTime.UtcNow, 2000);
+            load.VersionNumber.Should().Be(1);
 
             //Current State data not changed
-            response.CurrentState.CreatedAt.Should().Be(originalProcess.CurrentState.CreatedAt);
-            response.CurrentState.State.Should().BeEquivalentTo(originalProcess.CurrentState.State);
-            response.CurrentState.PermittedTriggers.Should().BeEquivalentTo(originalProcess.CurrentState.PermittedTriggers);
+            load.CurrentState.CreatedAt.Should().Be(originalProcess.CurrentState.CreatedAt);
+            load.CurrentState.State.Should().BeEquivalentTo(originalProcess.CurrentState.State);
+            load.CurrentState.PermittedTriggers.Should().BeEquivalentTo(originalProcess.CurrentState.PermittedTriggers);
 
             //Rest of the object should remain the same
-            response.Should().BeEquivalentTo(originalProcess, config => config.Excluding(x => x.CurrentState).Excluding(x => x.VersionNumber));
-            _logger.VerifyExact(LogLevel.Debug, $"Calling IDynamoDBContext.SaveAsync to update id {updatedProcessQuery.Id}", Times.Never());
+            load.Should().BeEquivalentTo(originalProcess, config => config.Excluding(x => x.CurrentState).Excluding(x => x.VersionNumber));
+            _logger.VerifyExact(LogLevel.Debug, $"Calling IDynamoDBContext.SaveAsync to update id {updatedProcessQuery.Id}", Times.Once());
+
+            //OldValues should be same as OrignalProcess
+            response.OldValues["assignment"].Should().BeEquivalentTo(originalProcess.ToDatabase().CurrentState.Assignment);
+            response.OldValues["formData"].Should().BeEquivalentTo(originalProcess.ToDatabase().CurrentState.ProcessData.FormData);
+            response.OldValues["documents"].Should().BeEquivalentTo(originalProcess.ToDatabase().CurrentState.ProcessData.Documents);
+
+            //NewValues should be same as UpdatedProcess
+            response.NewValues["assignment"].Should().BeEquivalentTo(response.UpdatedEntity.Assignment);
+            response.NewValues["formData"].Should().BeEquivalentTo(updatedProcess.CurrentState.ProcessData.FormData);
+            response.NewValues["documents"].Should().BeEquivalentTo(response.UpdatedEntity.ProcessData.Documents);
 
```

Comment:
I don't think you need all of these assertions - because you're mocking the entity updater, you don't need to check all of the values.